dbos/_dbos.py

```diff
@@ -974,6 +975,18 @@ async def get_event_async(
             lambda: DBOS.get_event(workflow_id, key, timeout_seconds)
         )
 
+    @classmethod
+    def get_all_events(cls, workflow_id: str) -> Dict[str, Any]:
```

Comment:
Let's put the docstring directly under this method, like `get_event` and other methods.
```
"""
        Get all events currently present for a workflow ID.
        Args:
            workflow_id: The workflow UUID to get events for
        Returns:
            A dictionary mapping event keys to their deserialized values
        """
```